Userland/Libraries/LibJIT/ELFBuild.h

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2023, Jesús Lapastora <cyber.gsuscode@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+// Collection of utilities to produce an in-memory ELF file in the same format
```

Comment:
This seems like we should move it to LibELF at some point.

Reply:
Managed to do it by removing the few GDB-specific things that remained. FYI I've changed `build_elf_image` so that it now accepts the `e_type` field as an argument, and the namespace names for the utilities.